paf-mvp-core-js/src/signing-definition/transmission-result-signing-definition.ts

```diff
@@ -0,0 +1,29 @@
+import { SIGN_SEP, SigningDefinition } from '@onekey/core/signing-definition/signing-definition';
+import { Seed, TransmissionResult } from '@onekey/core/model';
+
+export interface TransmissionResultSignatureData {
+  transmissionResult: TransmissionResult;
+  seed: Seed;
+}
+
+export class TransmissionResultSigningDefinition implements SigningDefinition<TransmissionResultSignatureData> {
```

Comment:
this definition doesnt seem to match the specification at https://github.com/OneKey-Network/addressability-framework/blob/main/mvp-spec/model/transmission-result.md which is:
```
status                  + '\u2063'
source.domain           + '\u2063' +
source.timestamp        + '\u2063' +
seed.source.signature+ '\u2063' +
contents[0].transaction_ids + '\u2063' +
contents[0].content_id + '\u2063' +
... + '\u2063' +
contents[n].transaction_ids + '\u2063' +
contents[n].content_id
```

also, wont transmissionResult.receiver and transmissionResult.source.domain always match anyway?

Reply:
Hi Brian, happy to get a review from you!

It's a good catch, unfortunately... this PR is correct, the documentation was wrong. I discovered it by trying to verify a real transmission that had been signed by Criteo.
In the file you linked, a nasty typo hides the first line of the content to sign 🤦‍♂️
I have pushed a fix for the typo yesterday [here](https://github.com/OneKey-Network/addressability-framework/pull/198).
I'm really sorry to discover this so late and I guess on your side you have implemented what was visible in the specification? (that would be logic!)
In that case, it would have to be updated 😳

As for your question
> also, wont transmissionResult.receiver and transmissionResult.source.domain always match anyway?

I suppose in most cases it would be true, but I assume @RomainLofaso wanted to have the possibility that an extra actor signs the transmission result, not necessarily the receiver (I can't recall).

Again, sorry for the confusion.

Reply:
@bwschmidt, I'll have to merge this PR now, as I'll be off for one week.
But happy to discuss further if you have more concerns.